content/en-us/reference/engine/classes/Accoutrement.yaml

```diff
@@ -5,23 +5,52 @@ memory_category: Instances
-  part using the `Class.Accoutrement.AttachmentPos|AttachmentPos`,
-  `Class.Accoutrement.AttachmentRight|Right`,
-  `Class.Accoutrement.AttachmentForward|Forward`, and
-  `Class.Accoutrement.AttachmentUp|Up` properties.
+  An `Class.Accoutrement` welds its child `Class.BasePart` called "Handle" to the
+  player's character. You can change the position and rotation of the "Handle"
+  part using either the `Class.Accoutrement.AttachmentPoint|AttachmentPoint` property or
+  an `Class.Attachment` within the handle.
 
-  Parts descending from an accoutrement are massless when attach to other parts
-  (e.g. with a Weld) as long as they are not the root part of the assembly that
-  `Class.BasePart:GetRootPart()|GetRootPart()` returns.
-  `Class.BasePart:GetMass()|GetMass()` returns 0 for parts in this case, and it
-  doesn't add to the total mass or rotational inertia of the Assembly.
+  When an `Class.Accoutrement` is attached to a character model, it will attach to the "Head" part by default.
+  A `Class.Weld` named "HeadWeld" will be created in the head with its `Class.Weld.Part0|Part0`
+  set to the Head and its `Class.Weld.Part1|Part1` set to the Handle. Its `Class.Weld.C0|C0`
+  will be set to `0, 0.5, 0` and its `Class.Weld.C1|C1` will be set to the Accoutrement's.
+  `Class.Accoutrement.AttachmentPoint`. The `Class.Weld.C1` will update automatically when the
+  `Class.Accoutrement.AttachmentPoint` changes. The HeadWeld will automatically
+  be destroyed if the Accoutrement is removed from the character.
 
-  This doesn't apply to a part descending from an accoutrement when an
-  accoutrement welds to another part that is massless or one if its parts
-  otherwise becomes root. This also doesn't apply for the root part, and it has
-  mass like a normal part.
+  The default attachment ("HeadWeld" behavior) can be overridden using a child
+  `Class.Attachment|attachment` within the Handle. If the Handle contains an
+  `Class.Attachment|attachment` wth a name matching the name of an `Class.Attachment|attachment`
+  somewhere in the character, it will be attached there instead of
+  being attached to the Head. When this happens, a `Class.Weld` named "AccessoryWeld"
+  will be automatically created within the Handle. It's `Class.Weld.Part0|Part0` will
+  be set to the Handle and its `Class.Weld.Part1|Part1` will be set to the
+  attached `Class.BasePart|part`. The AccessoryWeld's `Class.Weld.C0|C0` will be
+  set to the `Class.Attachment.CFrame|CFrame` of the Handle's `Class.Attachment`
+  and the `Class.Weld.C1|C1` will be set to the `Class.Attachment.CFrame|CFrame` of
+  the character `Class.Attachment|attachment`. These `Class.Weld.C0|C0` and `Class.Weld.C1|C1`
+  values will automatically update when either `Class.Attachment|attachment` is updated.
+  Like the Headweld, the AccessoryWeld will be destroyed automatically when
+  the Accoutrement is removed from the character.
```

Comment:
All of this is deeply technical behavior which may be interesting in some capacity, but what is the high-level takeaway that creators and scripters need to know here? Can you shorten this up to just the essential and practical aspects of how accoutrements work?